SU2_CFD/include/solvers/CEulerSolver.hpp

```diff
@@ -264,16 +264,18 @@ class CEulerSolver : public CSolver {
 
 #ifdef HAVE_OMP
   vector<GridColor<> > EdgeColoring;   /*!< \brief Edge colors. */
+  bool ReducerStrategy = false;        /*!< \brief If the reducer strategy is in use. */
 #else
   array<DummyGridColor<>,1> EdgeColoring;
+  /*--- Never use the reducer strategy if compiling for MPI-only. ---*/
+  static constexpr bool ReducerStrategy = false;
```

Comment:
The idea of `constexpr` here is to allow associated conditional statements to be optimized away by the compiler.